extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -697,6 +698,9 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     const ard = [];
     let exclusions;
     rtcResponseArray.forEach(rtcResponse => {
+      if (!rtcResponse) {
```

Comment:
What is an `rtcRepsonseDef`? Who constructs this array?

Reply:
An rtcResponseDef is the format of each Real Time Config response object that is built as part of of the Real Time Config flow, see [here](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/0.1/real-time-config-manager.js#L178) for where the array is built.

The issue arose from the fact that when the ad flow is cancelled, an in-progress RTC has its response set to undefined. Then, when we get into mergeRtcResponses, the checkStillCurrent above detects we are no longer in a current flow, and aborts. The underlying problem was that I mitsakenly had verifyStillCurrent() being called, which returns a closure, rather than actually verifying that the ad flow is still active (it should be changed to be called stillCurrentVerifierFactory or something like that). 

This additional check here is probably not needed anymore with the above fix, but I am adding it to be extra safe.

Reply:
See here for the def of rtcResponseDef 

https://github.com/ampproject/amphtml/blob/17ca7c219af01924d4a60ce4c3c2e247c86e5d85/build-system/amp.extern.js#L456